### ISSUE TITLE
Screen reader -- focus jumping around when sending/reading public chat

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#20697]
**Describe the bug**
Reported by Michael in https://groups.google.com/g/bigbluebutton-users/c/e3pCjZxY0Ac/m/IQb2JPOvAQAJ

> yesterday I had a large meeting with round about 60 persons on my server, running BBB 2.79. All of us using screenreaders, because of our visual impairment. Many of us wrote messages in the public chat. I was the moderator of the meeting and couldN't read the chat-history on the website properly. When ever I or another user tried to navigate the chat-area, the focus got lost and the cursor was jumping around on the website. Many users could reproduce this behaviour, browser- and screenreader independent. Is it an accessibility-issue, or do we have to configure our screenreaders different? 

> Polls and shared notes weren't in use. People were investigating the UI, changing the status and writing into the chat. When reading the chat-history, the cursor jumped to the top of the page, everytime, a new chat-message was written. The workaround was copying the chat-history into a text file in order to get the time, reading all of the history, because that's impossible with a braille display or a speech-driven screenreader if the cursor is moving to the top of the page around within seconds.

(some investigation with screen reader is needed on our side)

**To Reproduce**

**Expected behavior**

**Actual behavior**

**Screenshots**

**BBB version:**
BigBlueButton 2.7.9

**Comment by KDSBrowne** [comment]
I tried to reproduce the issue using both NVDA and JAWS screen readers but was unable to replicate the behavior described. Noticed that more details were requested in the forum thread, but it appears there hasn't been a reply yet.
